lib/util.ts

```diff
@@ -23,6 +24,26 @@ export function checkInputsShape(inputs: Tensor[], ...expectedDimensions: number
   return true;
 }
 
+export class ArrayUtil {
+  /**
+   * Verifies if 2 input arrays contain the same elements.
+   * @param n1 Array 1
+   * @param n2 Array 2
+   * @returns Whether these 2 are equal
+   */
+  static arraysEqual(n1: ReadonlyArray<number>|TypedArray, n2: ReadonlyArray<number>|TypedArray) {
```

Comment:
>TypedArray [](start = 47, length = 10)

[yulong]: consider using ArrayLike instead of explicitly listing all supported types.

Reply:
ArrayLike is an empty object that needs custom implementations of array methods like length or comparators, also the input here is a ReadonlyArray which requires custom converter to ArrayLike. It seems less convenient to use ArrayLike in this case.